cli/lib/kontena/client.rb

```diff
@@ -44,9 +45,7 @@ def initialize(api_url, token = nil, options = {})
       uri = URI.parse(@api_url)
       @host = uri.host
 
-      @logger = Logger.new(ENV["DEBUG"] ? $stderr : $stdout)
-      @logger.level = ENV["DEBUG"].nil? ? Logger::INFO : Logger::DEBUG
-      @logger.progname = 'CLIENT'
+      @logger = Kontena.logger
```

Comment:
Would there be some nice way to set `@logger.progname = "CLIENT"` here, to avoid the progname repitition throughout the class?

It would need some method to return a logger, rather than just a singleton accessor... would it be worth making a mixin module for that?